domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/Lovreferanse.kt

```diff
@@ -24,8 +24,19 @@ enum class Lovreferanse(val lovverk: String, val paragraf: String, val ledd: Str
     INSTITUSJONSOPPHOLD("Tiltakspengeforskriften", "§9", "1", "Opphold i institusjon, fengsel mv."),
     PENSJONSINNTEKT("Tiltakspengeforskriften", "§7", "1", "Forholdet til andre ytelser"),
     LØNNSINNTEKT("Rundskriv om tiltakspenger", "§8", "1", "Forholdet til lønn"),
-    ETTERLØNNARBEIDSMARKEDSLOV("Arbeidsmarkedsloven", "§13", "1", "Ytelser til gjennomføring av arbeidsmarkedstiltak mv."),
+    ETTERLØNNARBEIDSMARKEDSLOV(
```

Comment:
Fikk en tanke, kunne vi nesta disse innover som loven?

Reply:
Jeg vet ikke egentlig helt hva disse er ment å brukes til en gang jeg, for å være ærlig.. :)

Reply:
Nå så sendes de til frontend og vises som tekst til saksbehandler. Vi bør nok og lagre de pga. etterlevelse. Burde også hatt med mer metadata som commit hash, lovversjon mer?